src/snowflake/snowpark/modin/plugin/_internal/resample_utils.py

```diff
@@ -671,119 +672,32 @@ def perform_asof_join_on_frame(
     # 2023-01-07 02:00:00   NaN
     # 2023-01-10 00:00:00     6
 
-    # We want to perform an ASOF JOIN of preserving_frame and referenced_frame. Here
-    # are the steps to take:
-
-    # 1. Construct right_frame using referenced_frame, which has a
-    # temporary column, interval_end_col, that holds the closest
-    # following timestamp to every value in __index__. The last value in
-    # interval_end_col is dummy value that represents the smallest or largest
-    # (e.g. bfill or ffill) possible date in Snowflake.
-    interval_end_pandas_label = "interval_end_col"
     interval_start_snowflake_quoted_identifier = (
```

Comment:
since there is no interval anymore, maybe rename this to just right_timecol_snowflake_quoted_identifier to be consistent with the other name left_timecol_xxx